media-libs/sdl-net/sdl-net-1.2.8-r2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit multilib-minimal toolchain-funcs
+
+MY_P="${P/sdl-/SDL_}"
+DESCRIPTION="Simple Direct Media Layer Network Support Library"
+HOMEPAGE="https://www.libsdl.org/projects/SDL_net/index.html"
+SRC_URI="https://www.libsdl.org/projects/SDL_net/release/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-solaris"
+IUSE="static-libs"
+
+RDEPEND="media-libs/libsdl[${MULTILIB_USEDEP}]"
+DEPEND="${RDEPEND}"
+
+multilib_src_configure() {
+	AS="$(tc-getAS)" ECONF_SOURCE="${S}" econf \
```

Comment:
(Un)fortunately i couldn't reproduce it either.

What i found out was that after `configuring` the package, the generated `libtool` script contains a line with `as=AS`. However, i couldn't find anywhere in the build that `AS` was used actually.
I also saw that the llvm toolchain was used and was thinking this bug happens in combination with llvm (thus my initial note that i think i fixed it)..
And with setting `AS="$(tc-getAS)"` i could fix the `libtool` file to set it to the correct system variable..

However, since you also couldn't reproduce the problem with the llvm toolchain, I'm happily remove the workaround - and sorry that i haven't explained it more in detail.